serverless/src/getContactInfo/__tests__/handler.test.js

```diff
@@ -71,4 +85,31 @@ describe('getContactInfo', () => {
 
     expect(response.statusCode).toEqual(500)
   })
+
+  test('responds correctly on CMR-ordering error', async () => {
+    dbTracker.on('query', (query, step) => {
+      if (step === 1) {
+        query.response([
+          {
+            urs_id: { mock: 'ursId' },
+            urs_profile: { mock: 'urs' }
+          }
+        ])
+      } else {
+        query.response(undefined)
+      }
+    })
+
+    jest.spyOn(getCmrPreferencesData, 'getCmrPreferencesData').mockResolvedValue({
+      status: 200,
+      errors:
```

Comment:
I think this errors needs to be within a body object. Add an `expect(response.body).toEqual(...)` to ensure the error is being returned correctly